### PR TITLE
Individual package READMEs

### PR DESCRIPTION
**Pull request by potiuk**
---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/providers/google/BACKPORT_README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+##  Backport Google Package limitations:
+
+* Kubernetes Pod operators imports are refactored to contrib.kubernetes
```
Comment (mik-laj) [inline airflow/providers/google/BACKPORT_README.md]: I don't understand that. Can you tell more about it?

File: airflow/providers/google/BACKPORT_README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+* BigQuery operator is removed until it is refactored.
```
Comment (turbaszek) [inline airflow/providers/google/BACKPORT_README.md]:
```suggestion
* BigQuery operators are removed until it is refactored.
```

Comment (turbaszek) [reply airflow/providers/google/BACKPORT_README.md]: Does this mean that when I install backport packages I will have no BQ operators?

Comment (mik-laj) [reply airflow/providers/google/BACKPORT_README.md]: This is the expected result. I think it is worth adding information that we recommend using airflow.contrib operators and that new ones will be released soon.

Comment (turbaszek) [reply airflow/providers/google/BACKPORT_README.md]: I agree about adding this info. Otherwise, as a user I would suspect that after installation I will have no BQ operators, including those from contrib.

File: airflow/providers/google/BACKPORT_README.md (hunk 1 above)
Comment (potiuk) [reply airflow/providers/google/BACKPORT_README.md]: Fixed. Please check if it 's better now :)

File: airflow/providers/google/BACKPORT_README.md (hunk 2 above)
Comment (potiuk) [reply airflow/providers/google/BACKPORT_README.md]: Updated. Please check :)

File: airflow/providers/google/BACKPORT_README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+##  Backport Google Package limitations:
+
+* Kubernetes Pod operators are imported directly from airflow.contrib.
```
Comment (mik-laj) [inline airflow/providers/google/BACKPORT_README.md]:
```suggestion
```
This is not true. We use GKEPodOperator. We never use KubernetesPodOperator.

File: airflow/providers/google/BACKPORT_README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+##  Backport Google Package limitations:
```
Comment (mik-laj) [inline airflow/providers/google/BACKPORT_README.md]: Can you add tables that compare Airflow 1.10 and Airflow 2?  It's best if it was a few tables - one table that describes new operators, the other with moved operators.

File: airflow/providers/google/BACKPORT_README.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+* Kubernetes Pod operators are imported directly from airflow.contrib.
+* BigQuery operators are removed until it is refactored. This is going to happen
+  soon and we recommend to use the 1.10 BigQuery operators from the airflow.contrib
+  package.
+* GKEStartPodOperator is removed as it uses functionality only available in 2.0.
```
Comment (mik-laj) [inline airflow/providers/google/BACKPORT_README.md]: What do you think about something similar to the following text?

> The following operators have not been released:
> * All operators for BigQuery service. 
> * GKEStartPodOperator for Kubernetes Engine service.
> 
> We recommend staying with the old versions yet. These operators will be released as soon as possible.

File: airflow/providers/google/BACKPORT_README.md (hunk 4 above)
Comment (potiuk) [reply airflow/providers/google/BACKPORT_README.md]: CAn you add it please? I think you have an automated way of doing it and I might be mistaken again.

File: airflow/providers/google/BACKPORT_README.md (hunk 5 above)
Comment (potiuk) [reply airflow/providers/google/BACKPORT_README.md]: Feel free to add it.

File: airflow/providers/google/BACKPORT_README.md (hunk 3 above)
Comment (potiuk) [reply airflow/providers/google/BACKPORT_README.md]: Please. Describe it as you think is best.

**Comment by potiuk** [comment]
@mik-laj -> feel free to add your own commit on top with the changes. Otherwise we will be  just going ping-pong.

File: airflow/providers/google/BACKPORT_README.md (hunk 4 above)
Comment (mik-laj) [reply airflow/providers/google/BACKPORT_README.md]: If you want, I can prepare these tables for you.

Comment (potiuk) [reply airflow/providers/google/BACKPORT_README.md]: Please!

**Comment by mik-laj** [comment]
@mik-laj I'm starting to work on it.

File: airflow/providers/google/BACKPORT_README.md [hunk 6]
```diff
@@ -0,0 +1,362 @@
...
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+## Changelog
+
+### v2020.XX.XX
```
Comment (mik-laj) [inline airflow/providers/google/BACKPORT_README.md:38]: @potiuk Can you look at it?

Comment (potiuk) [reply airflow/providers/google/BACKPORT_README.md:38]: Beautiful! Thank you!

**Comment by mik-laj** [comment]
My part is done.

**Comment by potiuk** [comment]
I will rebase to latest master and squash!

File: airflow/providers/google/BACKPORT_README.md [hunk 7]
```diff
@@ -0,0 +1,362 @@
...
+**Limitations**
+
+The following operators have not been released:
+
+ * All operators for BigQuery service.
+ * GKEStartPodOperator for Kubernetes Engine service.
+
+We recommend staying with the old versions yet. These operators will be released as soon as possible.
+
+**New integration**:
```
Comment (mik-laj) [inline airflow/providers/google/BACKPORT_README.md]:
```suggestion
**New operators**:
```

**Review by mik-laj — APPROVED** [review]

File: airflow/providers/google/BACKPORT_README.md (hunk 6 above)
Comment (mik-laj) [inline airflow/providers/google/BACKPORT_README.md:38]:
```suggestion
### v20.XX.XX
```

File: tests/test_core_to_contrib.py [hunk 8]
```diff
@@ -1389,6 +1389,51 @@
...
+    ),
+    (
+        'airflow.providers.google.cloud.operators.cloud_sql.CloudSQLPatchInstanceDatabaseOperator',
+        'airflow.contrib.operators.gcp_sql_operator.CloudSqlInstanceDatabasePatchOperator'
+    ),
+    (
+        'airflow.providers.google.cloud.operators.cloud_storage_transfer_service'
+        '.CloudDataTransferServiceS3ToGCSOperator',
+        'airflow.contrib.operators.s3_to_gcs_transfer_operator.CloudDataTransferServiceS3ToGCSOperator'
+    ),
```
Comment (turbaszek) [inline tests/test_core_to_contrib.py:1436]: Is this change related?

File: airflow/providers/google/BACKPORT_README.md (hunk 6 above)
Comment (mik-laj) [reply airflow/providers/google/BACKPORT_README.md:38]: The PIP project applies a similar convention.

File: tests/test_core_to_contrib.py (hunk 8 above)
Comment (mik-laj) [reply tests/test_core_to_contrib.py:1436]: We could move it to another PR, but I had to do it to generate a table  for README.md.

File: airflow/providers/google/BACKPORT_README.md (hunk 6 above)
Comment (potiuk) [reply airflow/providers/google/BACKPORT_README.md:38]: Yes. I prefer 2020 as well.

File: tests/test_core_to_contrib.py (hunk 8 above)
Comment (potiuk) [reply tests/test_core_to_contrib.py:1436]: Yep. That's what I understood as well. The docs is generated from the code so it's good to sync it.
